### DISCUSSION TITLE
[projects beta] needs a quick way to go back to the repository from a project view

### DISCUSSION THREAD

**Discussion by atn38** [community/community#14511] [Projects and Issues]
just tried it today and had the devil of a time trying to find my way back to the repo!

**Comment by keilw** [comment]
I noticed that, too, same for organization-level projects where you can only get back to the GH home but not your organization.

**Comment by tmr08c** [comment]
Do you think #10448 is this same problem? It looks like that thread collected a few Discussion threads and has a bit more upvotes. If it's the same problem, you may want to consider upvoting it to get more eye on it  👀 .
